generator/src/main/java/com/scottlogic/deg/generator/utils/FileUtils.java

```diff
@@ -1,22 +1,74 @@
 package com.scottlogic.deg.generator.utils;
 
+import com.scottlogic.deg.generator.outputs.targets.FileOutputTarget;
+import com.scottlogic.deg.generator.outputs.targets.OutputTarget;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.text.DecimalFormat;
 import java.util.Arrays;
 
 public class FileUtils {
     /**
      * Provides a decimal formatter for pre-padding numbers with just enough zeroes.
      * E.g. if there are 42 records in a dataset this will produce a formatter of "00"
+     *
      * @param numberOfDataSets Maximum number of data sets we will number
      * @return Decimal format comprised of n zeroes where n is the number of digits in the input integer.
      */
-    public static DecimalFormat getDecimalFormat(int numberOfDataSets)
-    {
-        int maxNumberOfDigits = (int)Math.ceil(Math.log10(numberOfDataSets));
+    public static DecimalFormat getDecimalFormat(int numberOfDataSets) {
+        int maxNumberOfDigits = (int) Math.ceil(Math.log10(numberOfDataSets));
 
         char[] zeroes = new char[maxNumberOfDigits];
         Arrays.fill(zeroes, '0');
 
         return new DecimalFormat(new String(zeroes));
     }
+
+
+    public boolean exists(OutputTarget target) {
```

Comment:
Can this take a Path object instead? I don't think this is the right place for determining OutputTarget logic